src/main/java/de/retest/web/PeerConverter.java

```diff
@@ -81,7 +81,14 @@ private WebElementPeer convertToPeer( final String path, final WebData webData )
 		return peer;
 	}
 
+	protected boolean isRoot( final String parentPath ) {
+		return parentPath == null;
+	}
+
 	static String getParentPath( final String path ) {
+		if ( path.lastIndexOf( '/' ) == -1 ) {
```

Comment:
-1 means it doesn't occur at all, so I assume we could use:

```suggestion
		if ( !path.contains( "/" ) ) {
```